django/thunderstore/community/models/package_listing.py

```diff
@@ -160,31 +160,55 @@ def build_audit_event(
 
     @transaction.atomic
     def reject(
-        self, agent: Optional[UserType], rejection_reason: str, is_system: bool = False
+        self,
+        agent: Optional[UserType],
+        rejection_reason: str,
+        is_system: bool = False,
+        internal_notes: Optional[str] = None,
     ):
         if is_system or self.can_user_manage_approval_status(agent):
             self.rejection_reason = rejection_reason
             self.review_status = PackageListingReviewStatus.rejected
-            self.save()
+            self.notes = internal_notes or self.notes
+            self.save(
+                update_fields=(
+                    "rejection_reason",
+                    "review_status",
+                    "notes",
+                )
+            )
+            message = "\n\n".join(filter(bool, (rejection_reason, internal_notes)))
```

Comment:
Just pondering what sort of stuff the moderators will write on the field they regards as **internal** notes, because maybe someday some community wants to make the audit log public. Then the notes won't be just internal anymore?